### DISCUSSION TITLE
Custom Models and Skin

### DISCUSSION THREAD

**Discussion by slayerzim** [diamond-gold/MiniBosses#40] [Q&A] [answered]
I didn't quite understand how to do it, I'm not able to run the PHP script, could you give me a more explanatory tutorial Sorry for my English, I'm using Google translator because I'm Brazilian

**Comment by diamond-gold** [comment]
What the problem preventing you from running the script?

**Reply by slayerzim** [reply]
I don't know which program or how to run this script, if you can help me with a more explained tutorial saying which program or if you use a program to run this script, and also how to use custom models, I would be very grateful, because your plugin will be very important in my rpg server

**Comment by diamond-gold** [comment, marked as answer]
Assuming you are using Windows
1. Create a folder in your server folder, lets say `skins`
2. Save the script in this folder as `skin.php`
3. Copy your png images to this folder
4. Open command prompt and run the command `cd "Path to skins folder"`
5. Execute the script `..\bin\php\php skin.php`

As for custom models, just input the json in config under skin -> GeometryData
You may want to minify the json using sites such as https://codebeautify.org/jsonminifier

**Reply by slayerzim** [reply]
How do I put the skin using hosting, my server is not stored on my pc but on a hosting, the commands are not working on the hosting console

**Reply by diamond-gold** [reply]
I'm not sure if your hosting provides access to the terminal, many don't.
I would suggest downloading php binaries from [here](https://github.com/pmmp/PHP-Binaries/releases/tag/php-8.0-latest) and running the commands on your PC

**Reply by slayerzim** [reply]
can you tell me if there is any site that generates these codes for the skin?

**Reply by diamond-gold** [reply]
Not that I know of
